api.yaml

```
@@ -1,102 +1,74 @@
 openapi: 3.0.0
 info:
   version: 1.0.0
-  title: JSON Placeholder API
-  description: See https://jsonplaceholder.typicode.com/
+  title: TMDB API
+  description: See https://api.themoviedb.org/3/
 paths:
-  /posts:
+  /movie/popular:
     get:
-      description: Returns all posts
-      tags: ["Posts"]
-      operationId: "getPosts"
-      responses:
-        "200":
-          description: Successful response
-          content:
-            "application/json":
-              schema:
-                $ref: "#/components/schemas/PostsList"
-
-  /posts/{id}:
-    get:
-      description: Returns a post by id
-      tags: ["Posts"]
-      operationId: "getPost"
+      description: Returns all movies
+      tags: ["Movies"]
+      operationId: "getMovies"
```

ini dibuat lebih deskriptif lagi kayaknya

```suggestion
      operationId: "getPopularMovies"
```